quiche/src/lib.rs

```diff
@@ -7113,9 +7113,16 @@ impl<F: BufFactory> Connection<F> {
             reset_stream_count_remote: self.reset_stream_remote_count,
             stopped_stream_count_remote: self.stopped_stream_remote_count,
             path_challenge_rx_count: self.path_challenge_rx_count,
+            bytes_in_flight_duration: self.bytes_in_flight_duration(),
         }
     }
 
+    fn bytes_in_flight_duration(&self) -> Duration {
+        self.paths.iter().fold(Duration::ZERO, |acc, (_, path)| {
+            acc + path.bytes_in_flight_duration()
```

Comment:
Seems like adding up the duration over multiple paths will give a confusing value here.

For instance if I have a connection that last 10 sec, and it used two paths with bytes in flight for 8 sec each. Does that mean the connection `bytes_in_flight_duration` is 16 sec?

Disclaimer: I'm not sure I understand how multipath work.